apps/camera/js/views/viewfinder.js

```diff
@@ -284,7 +284,7 @@ module.exports = View.extend({
 
   template: function() {
     return '<div class="viewfinder-frame js-frame">' +
-        '<div class="viewfinder-video-container js-video-container">' +
+        '<div class="viewfinder-video-container js-video-container" aria-hidden="true">' +
```

Comment:
There's a linter fail: `apps/camera/js/views/viewfinder.js | line 287, col 90, Line is too long.`

Lines can't be longer that 80 characters. Please wrap it :)